src/v0/destinations/gainsight_px/config.js

```diff
@@ -1,12 +1,27 @@
 const { getMappingConfig } = require('../../util');
 
 const BASE_ENDPOINT = 'https://api.aptrinsic.com/v1';
-const ENDPOINTS = {
-  USERS_ENDPOINT: `${BASE_ENDPOINT}/users`,
-  CUSTOM_EVENTS_ENDPOINT: `${BASE_ENDPOINT}/events/custom`,
-  ACCOUNTS_ENDPOINT: `${BASE_ENDPOINT}/accounts`,
+const BASE_EU_ENDPOINT = 'https://api-eu.aptrinsic.com/v1';
+const BASE_US2_ENDPOINT = 'https://api-us2.aptrinsic.com/v1';
+
+const getBaseEndpoint = (Config) => {
```

Comment:
Lets add unit test cases for this please

Reply:
done.